lib/grammar.js

```diff
@@ -268,6 +268,16 @@ var grammar = module.exports = {
         return (o.params) ? 'rid:%s %s %s' : 'rid:%s %s';
       }
     },
+    { //a=imageattr:97 send [x=800,y=640,sar=1.1,q=0.6] [x=480,y=320] recv [x=330,y=250]
+      //a=imageattr:* send [x=800,y=640] recv *
+      //a=imageattr:100 recv [x=320,y=240]
+      push: 'imageattrs',
+      reg: /^imageattr:(\d+|\*)(?:[\s\t]+send[\s\t]+(\*|\[\S+\](?:[\s\t]+\[\S+\])?)+)?(?:[\s\t]+recv[\s\t]+(\*|\[\S+\](?:[\s\t]+\[\S+\])?)+)?/,
```

Comment:
yeah, you'd have to use that instead of literal, but they are both regexes, it should work fine. 

that said, it's not that bad, it just looks big at first sight due to all the repeated `[\s\t]` whitespace matchers. feel free to merge this now that you added the other changes.